infra/main.tf

```diff
@@ -17,7 +17,8 @@
 locals {
   unique = "${var.deployment_name}-${random_id.default.hex}"
 
-  application_image = "us-docker.pkg.dev/hsa-public/containers/cloud-client-api/${lower(var.language)}:${var.image_version}"
+  temp_image_version = "${var.image_version}-pr30"
```

Comment:
**Question:**
For now, if we make Java/Python/JavaScript source code changes, we use `temp_image_version` to test those changes.
Before merging, we would revert to `var.image_version`.
Future: We'll think of a solution for this chicken-and-egg `temp_image_version` problem and remove `temp_image_version` entirely.
Are my assumptions correct?

If so, this sounds good to me. :)
Again, thanks for being so thorough.

Reply:
I'll ping you internal docs on how this is intended to work